portingdb/load_data.py

```diff
@@ -330,3 +330,8 @@ def load_from_directories(data, directories):
                if names:
                     nonpy_requirers.setdefault(kind, set()).update(names)
         package['non_python_requirers'] = nonpy_requirers
+
+    for name, isodt in data_from_file(directories, 'orphans').items():
+        if name in packages:
+            last_change = datetime.datetime.fromisoformat(isodt)
+            packages[name]['last_repo_metadata_change'] = last_change
```

Comment:
In case of else, you can assume the package is orphaned but not yet in orphans.json, hence it was orphaned "now".
(Assuming the property is only added to packages that are orphaned, or at least ignored otherwise)

the only other way I can think of is that it was already retired and the repo mirror used to update portingdb doesn't know it yet, but the one generating the orphans.json knew it. I find it highly unlikely (orphans.json takes copule hours to be published) and the data for the retired package would be wrong anyway.

Reply:
The else is mainly for non-Python packages.

But in any of the 3 cases (mine & 2 yours), we should be fine having no data and showing no hourglass, right?

Reply:
right